src/app/zap-templates/zcl/data-model/chip/time-format-localization-cluster.xml

```diff
@@ -48,7 +48,7 @@ limitations under the License.
       or audibly convey time information need a mechanism by which they can be configured to use a
       user’s preferred format.</description>
     <!-- Base data types -->
-    <attribute side="server" code="0x00" define="HOUR_FORMAT" type="ENUM8" writable="true" optional="false">HourFormat</attribute>
+    <attribute side="server" code="0x00" define="HOUR_FORMAT" type="ENUM8" min="0x00" max="0x01" writable="true" optional="false">HourFormat</attribute>
```

Comment:
```suggestion
    <attribute side="server" code="0x00" define="HOUR_FORMAT" type="HourFormat" min="0x00" max="0x01" writable="true" optional="false">HourFormat</attribute>
```

Reply:
type="HourFormat" min="0x00" max="0x01" will cause the following compile error in the auto generated code.

```
../../../examples/lighting-app/linux/third_party/connectedhomeip/zzz_generated/app-common/app-common/zap-generated/cluster-objects.h:9134:98: error: ‘HourFormat’ does not name a type
 9134 |         Attributes::HourFormat::TypeInfo::DecodableType hourFormat                 = static_cast<HourFormat>(0);
```

Reply:
Fixed in https://github.com/project-chip/connectedhomeip/pull/13744.